cs-cz/javascript.html.markdown

```diff
@@ -9,32 +9,27 @@ lang: cs-cz
 filename: javascript-cz.js
 ---
```

Comment:
Wouldn't you like to add yourself to the translators list?

Reply:
No, I am OK with just being in git commit history.